include/deal.II/base/mpi_compute_index_owner_internal.h

```diff
@@ -34,6 +34,98 @@ namespace Utilities
        */
       namespace ComputeIndexOwner
       {
+        class DictionaryPayLoad
```

Comment:
can you add a short doxygen comment here so that it appears in the documentation please (and fix the comment 4 lines up)?